### DISCUSSION TITLE
How to compile the external?

### DISCUSSION THREAD

**Discussion by kWAYTV** [IMXNOOBX/cs2-external-esp#108] [Q&A]
How to compile the external?

**Comment by IMXNOOBX** [comment]
Build the program using Visual Studio 2022

Build: x64 - Release
Locate your binary file in the folder <arch>/<configuration>, e.g., x64/Release.

❕ In case the offsets get outdated (Every game update), you could check UnnamedZ03's repository for the updated ones [here](https://github.com/UnnamedZ03/CS2-external-base/blob/58466cd7feba2fbcf5ab49b0dbbdc7bcd6d7df58/source/CSSPlayer.hpp#L3-L15)

**Comment by ifBars** [comment]
> How to compile the external?

[https://www.youtube.com/watch?v=meoZczd1urE](https://www.youtube.com/watch?v=meoZczd1urE) Remember to use OP's repo to compile

You can also use A2X's CS2 Dumper to dump offsets yourself, remember to load your game without VAC using -insecure, when dumping offsets. [https://github.com/a2x/cs2-dumper](https://github.com/a2x/cs2-dumper)
